library/common.py

```diff
@@ -70,3 +71,30 @@ def generate_text(seed_text, next_words, model, max_sequence_len):
                 break
         seed_text += " "+output_word
     return seed_text.title()
+
+
+def _random_prefix(sentences):
+    """
+    prefix random generator
+    input: list of input sentences
+    output: random word
+    """
+
+    words = _word_dict(sentences)
+    return choice(words)
+
+def _word_dict(sentences):
+    """
+    input: list of input sentences
+    output: unique list of the corpus
+    #to-do: strip out punctuation
```

Comment:
I _think_ we could do something like `sentences.translate(None, string.punctuation)` prior to pulling out the unique values per this [stackoverflow](Do you think we could use https://stackoverflow.com/questions/265960/best-way-to-strip-punctuation-from-a-string).